src/Sia.Gateway/Initialization/Startup/AutoMapperStartup.cs

```diff
@@ -38,9 +39,21 @@ public static void InitializeAutomapper()
                     .UseResolveJsonToString();
                 configuration.CreateMap<Event, Data.Incidents.Models.Event>().EqualityById()
                     .UseResolveJsonToString();
-                configuration.CreateMap<Data.Incidents.Models.Event, Event>().EqualityById()
+                configuration.CreateMap<Data.Incidents.Models.Event, Event>()
+                    .ForMember
+                    (
+                        nameof(Event.PrimaryTicketId),
+                        options => options.MapFrom(
+                            (src) => src.Incident == null
```

Comment:
This ternary chain is less than ideal; if we were allowed to use the null-coalesce operator in statements executed by Linq's query provider, we would use that syntax rather than this one; it may make sense to add a comment explaining that. Something like 

// null coalesce operator disallowed in mapping statements, ternary used as alternative

could prevent some of our future selves' confusion over why this was done this way.

The null coalesce way to replace these 8 lines of ternary operator is much, much cleaner:
src?.Incident?.Tickets?.FirstOrDefault(ticket => ticket.IsPrimary)?.OriginId
If only we could use it here.